pkg/interface/src/components/OmniboxResult.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Row, Icon, Text } from '@tlon/indigo-react';
+import defaultApps from '../lib/default-apps';
+
+export class OmniboxResult extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSelected: false,
+      hovered: false
+    };
+    this.setHover = this.setHover.bind(this);
+  }
+
+  setHover(boolean) {
+    this.setState({ hovered: boolean });
+  }
+  render() {
+    const { icon, text, subtext, link, navigate, selected, dark } = this.props;
+
+    let invertGraphic = {};
+
+    if (icon.toLowerCase() !== 'dojo') {
+      invertGraphic = (!dark && this.state.hovered) ||
+      selected === link ||
+      (dark && !(this.state.hovered || selected === link))
+        ? { filter: "invert(1)" }
```

Comment:
we do this a lot, @g-a-v-i-n can we put this in indigo-react sometime.

Reply:
This stuff is just because Dojo breaks the pattern. What we want is to put the app icons into `<Icon>` and then dispose of all the image stuff and invert patterns, because it handles the fill for us.

Reply:
oh sweet.